src/references/script_debugger.md

```diff
@@ -0,0 +1,97 @@
+---
+id: script_debugger
+name: Script Debugger Reference
+title: Script Debugger Reference
+tags:
+    - Reference
+---
+
+# Script Debugger
+
+## Overview
+
+The **Script Debugger** can help narrow down which lines of code have issues or bugs by allowing creators to pause script execution and move specifically through different lines of **Core** Lua scripts. Spending time looking for issues or bugs in a script can be very time consuming. This is where the **Script Debugger** in **Core** can help narrow down where in the script there are issues or bugs.
+
+A common way to debug a game to find issues or bugs, is to use the `print` function to print out values to the **Event Log**. An advantage of using the **Script Debugger** is the whole game state can be debugged. When using the `print` method, values that get printed, need to be thought about in advanced.
+
+## Opening the Script Debugger
+
+From the **Window** menu, select **Script Debugger** to open up the **Script Debugger** window.
+
+!!! info "Script Debugger and External Editors"
+    The **Script Debugger** only works with the **Script Editor** in **Core**, and not external editors.
+
+![Opening](../img/ScriptDebugger/opening_script_debugger.png)
+![Window](../img/ScriptDebugger/script_debugger_window.png)
+{: .image-cluster}
+
+## Enabling the Script Debugger
+
+The **Script Debugger** needs to be enabled so any breakpoints added, will pause the execution of the game. This allows areas of interest in a script to be inspected that may contain an issue.
+
+Clicking on the ![Disabled](../img/ScriptDebugger/script_debugger_disabled.png)icon will enable the **Script Debugger**. When enabled, the icon will turn red ![Enabled](../img/ScriptDebugger/script_debugger_enabled.png), indicating the **Script Debugger** is enabled. With the **Script Debugger** now enabled, any breakpoints added will pause execution of the game.
+{: .image-inline-text .image-background }
+
+!!! info "When disabling the **Script Debugger**, all breakpoints will also be disabled. This is useful, because all breakpoints don't need to be removed. Next time the **Script Debugger** is enabled, the breakpoints will also be enabled."
```

Comment:
This is useful, because all breakpoints don't need to be removed > This allows creators to disable all breakpoints without removing them